### ISSUE TITLE
question

### ISSUE THREAD

**Issue by Daveied** [BinBashBanana/webretro#75]
is there a way how i can set it up with my account name? such as it being https://daveied.github.io/webretro/ just incase my school blocks your website i can just fork it and make my own website from there

**Comment by foxsouns** [comment]
Yes: I'm in a hurry to catch a bus right now, but if you'll give me 20 minutes I can write the instructions down.

**Comment by foxsouns** [comment]
1. fork the repository
2. open the `Settings` tab
3. scroll down to `Code and automation > Pages`
4. select `Deploy from branch`, select branch (usually `master`), folder `root`, and `save`
5. wait for github to deploy the site (no more than 3 minutes, usually)

access your site from `<user>.github.io/<repository>/`

**Comment by BinBashBanana** [comment]
^
